store/storetest/channel_store.go

```diff
@@ -57,23 +58,23 @@ func testChannelStoreSave(t *testing.T, ss store.Store) {
 	o1.Name = "zz" + model.NewId() + "b"
 	o1.Type = model.CHANNEL_OPEN
 
-	if err := (<-ss.Channel().Save(&o1)).Err; err != nil {
+	if err := (<-ss.Channel().Save(&o1, -1)).Err; err != nil {
```

Comment:
For the tests where we don't want or care about a limit, pass `-1` to bypass the limit check and speed up the tests.